build/root/usr/local/bin/run

```diff
@@ -74,7 +74,11 @@ else
     echo "Using '$ARTIFACT_DIR' to store the test artifacts"
 fi
 
-if [ $1 == "gpu-operator_test-master-branch" ]; then
+if [ $1 == "cluster-upgrade" ]; then
+    target="cluster"
+    action="upgrade"
+    shift 1
+elif [ $1 == "gpu-operator_test-master-branch" ]; then
```

Comment:
unnecessary move for this PR

Reply:
maybe you can remove this temporary hack (single name -> `target` + `action`) in a dedicated commit, as part of this PR

Reply:
was not a hack, I just ordered what was there by alphabetic order... a little OCD on my end

Reply:
+1 for the OCD part,

but "hack" is what is below, what we put with David a few weeks ago, so that
```
run cluster upgrade
```
and 
```
run cluster-upgrade
```
have the same behavior, with the help of this block
```
if [ $1 == "cluster-upgrade" ]; then
            target="cluster"
            action="upgrade"
```

but this "single argument" entrypoint isn't needed anymore as Prow has taken into account the update

Reply:
ok restored to it's original version, I do still think we should follow patterns, as alphabetical order.... but something for the future